inspirehep/utils/url.py

```diff
@@ -89,3 +90,17 @@ def retrieve_uri(uri, outdir=None):
 
         local_file.flush()
         yield local_file.name
+
+
+def get_prod_url_for_recid(recid, pattern_config_var='LEGACY_RECORD_URL_PATTERN'):
```

Comment:
And why not having `pattern` as argument directly rather than the name of a config var?
So the call would be:  `get_prod_url_for_recid(recid, pattern=current_app.config[pattern_config_var])`

Reply:
We realized that in this case the entire function would have no sense, thus we hardcode the pattern in the function and rename it to `get_legacy_url_for_recid()`.